starboard/input.h

```diff
@@ -74,6 +74,11 @@ typedef enum SbInputDeviceType {
   //
   // Produces |Input| events.
   kSbInputDeviceTypeOnScreenKeyboard,
+
+  // Input from a gyroscope
+  //
+  // Produces |Input| event
+  kSbInputDeviceGyroSensor,
```

Comment:
@y4vor WDYT about this addition to Starboard?

Reply:
Please guard all API changes with the SB_API_VERSION checks. Also update starboard/CHANGELOG.md.

Here are the instructions of how to introduce a Starboard API change:
https://g3doc.corp.google.com/company/teams/cobalt/team/components/starboard.md#make-changes-in-trunk-to-the-latest-unreleased-starboard-version1

Reply:
We don't have access to this doc. Could you share it with us via docs.google.com?